apis/compute/v1beta1/firewallpolicyrule_types.go

```diff
@@ -102,12 +102,14 @@ type ComputeFirewallPolicyRuleSpec struct {
 	// +optional
 	EnableLogging *bool `json:"enableLogging,omitempty"`
 
+	// +kubebuilder:validation:XValidation:rule="self == oldSelf",message="the field is immutable"
```

Comment:
These fields are related to identity. We do not need the immutable check, but set this as required.